beater/beater.go

```diff
@@ -7,14 +7,18 @@ import (
 	"net/http"
 	"regexp"
 
+	"sync"
```

Comment:
I always have this problem, this is how it looks after running `gofmt` and `goimports`. Eventually got tired of fixing white lines in import blocks manually... it might be because of my IDE, but `make check` seems to be happy with this indeed.